### ISSUE TITLE
Load database from subdirectory

### ISSUE THREAD

**Issue by paulequilibrio** [hesstobi/atom-autocomplete-glossaries#4]
It's possible to load the `.glsdefs` file from a subdirectory?
I'm using `latexmk` and setting `$out_dir` variable to a subdirectory, but this package seems not found the `.glsdefs` file.

**Comment by hesstobi** [comment]
Yes this should be possible. Nevertheless, I don't have time to look in this right now. 
Maybe  you can prepare a pull request. Look at these lines in the source https://github.com/hesstobi/atom-autocomplete-glossaries/blob/e69dcadd407bc1035d25a2aea568d810418ceb8a/lib/label-manager.coffee#L43
